SSA/Projects/LLVMRiscV/Pipeline/const.lean

```diff
@@ -0,0 +1,2233 @@
+-- AUTOGENERATED Lean file: Constant lowering for RISC-V li instructions
```

Comment:
Good thinking to put this comment here! For completeness sake, you might also add a comment that explicitly points to which script was used to generate it, or with instructions in case someone else wanted to tweak/regenerate the lowerings